google-cloud-spanner/clirr-ignored-differences.xml

```diff
@@ -35,6 +35,46 @@
     <className>com/google/cloud/spanner/connection/ConnectionOptions</className>
     <method>com.google.cloud.spanner.Dialect getDialect()</method>
   </difference>
+  <difference>
+    <differenceType>7004</differenceType>
+    <className>com/google/cloud/spanner/InstanceConfig</className>
+    <method>InstanceConfig(com.google.cloud.spanner.InstanceConfigId, java.lang.String, java.util.List, java.util.List, com.google.cloud.spanner.InstanceAdminClient)</method>
+  </difference>
+  <difference>
+    <differenceType>7004</differenceType>
+    <className>com/google/cloud/spanner/InstanceConfigInfo</className>
+    <method>InstanceConfigInfo(com.google.cloud.spanner.InstanceConfigId, java.lang.String, java.util.List, java.util.List)</method>
```

Comment:
I don't think we can have breaking changes in constructors, could you please overload it instead of changing it to avoid the breakage?

Reply:
This constructor has not been changed, and these clirr differences are not required here. Will remove these. Thanks for pointing this out.

Reply:
Removed these differences.